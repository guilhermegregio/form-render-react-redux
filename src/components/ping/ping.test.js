import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';

import Ping from './ping';
import configureStore from '../../store/store';

const store = configureStore();

it('default render app component', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Provider store={store}><Ping /></Provider>, div);

  expect(div).toMatchSnapshot();
});
