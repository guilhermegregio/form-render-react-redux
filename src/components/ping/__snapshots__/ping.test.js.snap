// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default render app component 1`] = `
<div>
  <div
    data-reactroot=""
  >
    <h2>
      <!-- react-text: 3 -->
      is pinging: 
      <!-- /react-text -->
      <!-- react-text: 4 -->
      false
      <!-- /react-text -->
    </h2>
    <button>
      Start PING
    </button>
    <p>
      Teste
    </p>
  </div>
</div>
`;
